pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies = [
     "virtualenv>=20",
     "pyproject-hooks",
     "requests-toolbelt",
-    "unearth>=0.12.1",
+    "unearth==0.14.0",
```

Comment:
please change it to a minimum constraint instead of an exact match. or is there any reason behind this?

Reply:
Fixed :)